src/react-apps/applications/ux-editor/src/actions/formDesignerActions/actions/addApplicationMetadata/index.ts

```diff
@@ -0,0 +1,62 @@
+import { Action } from 'redux';
+import * as ActionTypes from '../../formDesignerActionTypes';
+
+export interface IAddApplicationMetadataAction extends Action {
+  id: string,
+  maxFiles: number;
+  maxSize: number;
+  fileType: string;
+  callback?: (...args: any[]) => any;
+}
+
+export interface IAddApplicationMetadataActionFulfilled extends Action {
```

Comment:
Are all these types necesarry? You don't handle the fulfilled in the reducer?

Reply:
Fixed